cclib/parser/gaussianparser.py

```diff
@@ -1554,6 +1554,74 @@ def extract(self, inputfile, line):
                 self.nocoeffs = nocoeffs
                 self.nooccnos = nooccnos
 
+
+        # Natural spin orbital coefficients (nsocoeffs) and occupation numbers (nsooccnos)
+        if line[5:39] == "Alpha Natural Orbital Coefficients" or line[5:38] == "Beta Natural Orbital Coefficients":
+            if line[5:38] == "Beta Natural Orbital Coefficients":
+                beta = True
+                self.nsocoeffs.append(numpy.zeros((self.nmo, self.nbasis), "d"))
+                self.nsooccnos.append(list())
+            else:
+              beta = False
```

Comment:
What's with the indent here in the `else` block? Looks like only 2 spaces compared with 4 elsewhere.

Reply:
Looks like the else is now gone.